README.md

```diff
@@ -26,25 +25,28 @@ and then install the package
 
 ## Supported hypervisors
 
-virt-who can obtain list of guest running on given machine or association between hosts and guests in given virtualization environment.
+virt-who can obtain list of guest running on given machine or association between hosts and guests in given
+virtualization environment.
 
 Backends for reporting list of guest running on given machine:
+
 * libvirt
 
 Backends for reporting hosts to guests association:
+
 * RHV-M (4.x)
```

Comment:
hmm wasn't this removed by #384?

Reply:
I think that the latest version used libvirt anyway, but I think that we can remove it from this doc, because: https://access.redhat.com/support/policy/updates/rhev